README.md

```diff
@@ -1,4 +1,24 @@
 # gplibrary
-Useful subsystem models
+
+This repository includes a number of useful GP- and SP- compatible models. The current list includes:
+
+* **GP models**
+  * Aircraft
+    * Engine
+    * Fuselage
+    * Tail
+    * Wing
+    * Mission: a model that unifies the subsystems and the aircraft flight profile
+  * Beam: discretized beam for distributed loads
+* **SP models**
+  * SimpleAC: a basic aircraft model 
+  * Aircraft
+    * Tail
+    * Wing
+* **Miscellanous models**
+  * Economic Order Quantity
+  * Moment of Inertia
+  * Net Present Value
+  * Raymer Weights: rule-of-thumb weight relations for aircraft design
 
 [![Build Status](https://acdl.mit.edu/csi/job/gpkit_commons_Push_Models/badge/icon)](https://acdl.mit.edu/csi/job/gpkit_commons_Push_Models/)
```

Comment:
this should be at the top

Reply:
You can make edits directly on this branch using the browser!!

Reply:
Done :p